cmd/mindthegap/create/helmbundle/helm_bundle.go

```diff
@@ -250,5 +251,8 @@ func NewCommand(out output.Output) *cobra.Command {
 	cmd.Flags().
 		BoolVar(&overwrite, "overwrite", false, "Overwrite Helm charts bundle file if it already exists")
 
+	// TODO Unhide this from DKP CLI once DKP supports OCI registry for Helm charts.
+	utils.AddCmdAnnotation(cmd, "exclude-from-dkp-cli", "true")
```

Comment:
Just curious, why can we not set `cmd.Hidden`?

Reply:
Because I don't want them hidden when running mindthegap as a standalone binary, only when embedding in the DKP CLI because mindthegap supports OCI registries for helm chart storage, but that isn't supported in DKP CLI yet...

Reply:
Got it, thanks!